ConfigurationSystem/Client/Helpers/CSGlobals.py

```diff
@@ -26,26 +75,13 @@ def getCSExtensions():
     Return list of extensions registered in the CS
     They do not include DIRAC
   """
-  from DIRAC import gConfig
-  return gConfig.getValue( '/DIRAC/Extensions', [] )
+  return Extensions().getCSExtensions()
```

Comment:
It is a backport from v7. That will consistently return the extension name (with DIRAC at the end) and also check that the extension exists.